### ISSUE TITLE
Points > Manage Action triggered only once

### ISSUE THREAD

**Issue by YannickBiet** [mautic/mautic#4582]
| Q  | A
| ---| ---
| Bug report? | Yes
| Feature request? | No
| Enhancement? | No

## Description:
To my knwoledge, points actions should be configured a way to let mautic admin says we want to modify contact's score when a contact is doing a specific action.

For example, there's a simple functionnal example we always give to our users : 
- **each time** a contact visit a page on your website with mautic tracking code enabled, it would be great to add +1 to the contact's score

So we configure the following Score Action item :

![image](https://user-images.githubusercontent.com/930391/28956394-d991f618-78ec-11e7-9a8b-41ec6613180a.png)

When a contact comes on the website, it is added as anonymous contact + has the score initialized to 1

![image](https://user-images.githubusercontent.com/930391/28956425-20082a54-78ed-11e7-80dc-8293a404392c.png)

If i navigate to another page, the page is added to mautic timeline, BUT this time the actions does not trigger any score modification !

![image](https://user-images.githubusercontent.com/930391/28956440-3da51630-78ed-11e7-92f2-50119eae2145.png)

How can we easily setup score modifications if the action is only triggered once.

Can someone confirm this as a bug ?

## If a bug:

| Q   | A
| --- | ---
| Mautic version | 2.8.2 and reproduced in 2.9
| PHP version | 7.0.12

**Comment by escopecz** [comment]
Seems like duplicate of https://github.com/mautic/mautic/issues/929
